### ISSUE TITLE
Getting a quick sample of data from the artifacts

### ISSUE THREAD

**Issue by eu9ene** [mozilla/firefox-translations-training#405]
When inspecting the running pipeline I have to download artifacts on the local machine quite often. I essentially do:

`wget http://artifact.zst`
`zstd -dc artifact.zst | head -n 100` or similar. Would it be possible to integrate this in Taskcluster UI? Say, as an extra button near each artifact?

It will save a lot of time, especially for large datasets and we'll be able to inspect the data on different stages more often.

Alternatively we could just log data samples for each step.

**Comment by gregtatum** [comment]
I would like to have more samples of data in the artifacts just in general. It would be nice to have statistic about corpus size, how much was filtered, samples of data, etc.

**Comment by marco-c** [comment]
I agree. I would vote to do this in our scripts that run within the tasks, rather than at the Taskcluster level, because our needs are quite specific and different tasks will need different things, etc.
